data/approved_tags.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
as per the private repo comment:

Is this data folder also used for anything on the rest of the application? We probably shouldn't mix data used for unit tests and for the rest of the application.

This would probably make sense to on the spec folder since specs usually contain everything they need to run unless there's a good reason not to.
Such as a folder under spec called helper, data or resources.

Reply:
you're totally right, this was my instruction so I take the blame. @chowdud please create a data folder inside `/spec` as suggested by duarte.